app/src/main/res/layout/fragment_splash.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+             android:background="@color/colorPrimary"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+
+    <ImageView
+      android:layout_width="match_parent"
+      android:layout_height="130dp"
+      android:scaleType="fitXY"
+      android:adjustViewBounds="true"
+      android:layout_marginLeft="60dp"
+      android:layout_marginRight="60dp"
+      android:src="@drawable/ic_oppia_logo_white"
+      android:layout_gravity="center"
+      android:layout_marginBottom="100dp"/>
+
+</FrameLayout>
```

Comment:
Similar to above, please keep EOF line endings for consistency.

Reply:
done